src/main/java/it/polimi/telco/model/ServiceSubscription.java

```diff
@@ -0,0 +1,63 @@
+package it.polimi.telco.model;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.Date;
+import java.util.List;
+
+@Entity
+@Table(name = "service_subscription", schema = "telco")
+@NamedQuery(name = "ServiceSubscription.getServiceSubscriptionById", query = "SELECT ss FROM ServiceSubscription ss  WHERE ss.id = ?1 ")
+public class ServiceSubscription {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
```

Comment:
I think we're missing a relationship with the user (one subscription is associated with one user)